variegated-embassy-ads124s08/src/registers.rs

```diff
@@ -526,17 +798,38 @@ impl ByteRepresentation for InternalVoltageReferenceConfiguration {
 }
 
 #[derive(Debug, Clone, Copy, Default, Eq, PartialEq, Format)]
+/// IDACMagnitude
```

Comment:
This enum can be better explained, reference the data sheet to craft a better explanation.